src/posts/posts.11tyData.js

```diff
@@ -0,0 +1,18 @@
+const ALPHABETICAL_RE = new RegExp("[^a-zA-Z0-9 -]");
+
+module.exports = {
+	layout: "post",
+	eleventyComputed: {
+		eleventyNavigation: {
+			key: ({ category }) => category,
+			title: ({ title }) => title,
+		},
+		tags: ({ category, tags }) => {
+			if (tags) {
+				return tags;
+			} else {
+				return [category.replace(ALPHABETICAL_RE, "").toLowerCase()];
+			}
+		},
+	},
+};
```

Comment:
1️⃣ 🔍 

This JavaScript file sets the layout of every `post` template to `'post'`. It also reads the front matter-supplied `category` and `title` to set `eleventyNavigation` data.